### PR TITLE
Make pagination more screen-reader friendly

### PR DESCRIPTION
**Pull request by ksiwei**

### CODE REVIEW

File: packages/bpk-component-pagination/readme.md [hunk 1]
```diff
@@ -38,3 +39,5 @@ const Pagination = () => (
 | onPageChange      | func                 | false    | null          |
 | visibleRange      | number               | false    | 3             |
 | className         | string               | false    | null          |
+| paginationLabel   | string               | false    | "pagination navigation" |
```
Comment (k0nserv) [inline packages/bpk-component-pagination/readme.md]: These should be aligned. Use an online formatter

File: packages/bpk-component-pagination/src/BpkPaginationPage.js [hunk 2]
```diff
@@ -25,32 +25,41 @@ import STYLES from './bpk-pagination-page.scss';
...
-  label: PropTypes.node.isRequired,
+  page: PropTypes.number.isRequired,
   onSelect: PropTypes.func.isRequired,
   isSelected: PropTypes.bool,
+  pageLabelFormatter: PropTypes.func,
 };
 
 BpkPaginationPage.defaultProps = {
   isSelected: false,
+  pageLabelFormatter: defaultLabelFormatter,
```
Comment (k0nserv) [inline packages/bpk-component-pagination/src/BpkPaginationPage.js]: We don't typically provided a default for these as it makes it possible to miss the translation piece

File: packages/bpk-component-pagination/src/BpkPagination.js [hunk 3]
```diff
@@ -54,7 +56,12 @@ const BpkPagination = (props) => {
   const hasNextPage = selectedPageIndex !== pageCount - 1;
 
   return (
-    <section className={classNames.join('')} {...rest}>
+    <section
```
Comment (k0nserv) [inline packages/bpk-component-pagination/src/BpkPagination.js]: http://www.a11ymatters.com/pattern/pagination/ recommends making this a `nav`.

File: packages/bpk-component-pagination/src/BpkPagination.js [hunk 4]
```diff
@@ -84,12 +92,16 @@ BpkPagination.propTypes = {
...
   className: PropTypes.string,
+  paginationLabel: PropTypes.string,
+  pageLabelFormatter: PropTypes.func,
 };
 
 BpkPagination.defaultProps = {
   onPageChange: null,
   visibleRange: 3,
   className: null,
+  paginationLabel: 'Pagination navigation',
```
Comment (k0nserv) [inline packages/bpk-component-pagination/src/BpkPagination.js]: Don't provide a default for this and make  it required

File: packages/bpk-component-pagination/src/BpkPagination.js [hunk 5]
```diff
@@ -84,12 +92,16 @@ BpkPagination.propTypes = {
...
+  pageLabelFormatter: null,
```
Comment (k0nserv) [inline packages/bpk-component-pagination/src/BpkPagination.js]: Same as above

File: packages/bpk-component-pagination/readme.md [hunk 6]
```diff
@@ -35,6 +36,8 @@ const Pagination = () => (
 | selectedPageIndex | number               | true     | -             |
 | previousLabel     | string               | true     | -             |
 | nextLabel         | string               | true     | -             |
+| paginationLabel   | string               | true     | -             |
+| pageLabelFormatter| func                 | true     | -             |
```
Comment (matthewdavidson) [inline packages/bpk-component-pagination/readme.md]: can we rename `pageLabelFormatter` -> `pageLabel`?

I suspect you added `Formatter` to infer that its a func as opposed to a string - but thats what prop types are for.

cc @k0nserv

Comment (k0nserv) [reply packages/bpk-component-pagination/readme.md]: Agreed, will do

**Pull request merged**